docs/en/operations/settings/settings.md

```diff
@@ -2864,4 +2864,53 @@ Possible values:
 
 Default value: `0`.
 
+## limit {#limit}
+
+Sets the number of rows to get from the query result. It adjust the limit previously set by the [LIMIT](../../sql-reference/statements/select/limit.md#limit-clause) clause.
+
```

Comment:
```suggestion
Sets the maximum number of rows to get from the query result. It adjusts the value set by the [LIMIT](../../sql-reference/statements/select/limit.md#limit-clause) clause, so that the limit, set in the expression, cannot exceed the limit, set by this setting.

```